[2025-07-13T10:36:15.495+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-07-13T10:36:15.525+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: employee_ingestion_dag.split_and_save_data scheduled__2025-07-13T10:35:00+00:00 [queued]>
[2025-07-13T10:36:15.536+0000] {taskinstance.py:2614} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: employee_ingestion_dag.split_and_save_data scheduled__2025-07-13T10:35:00+00:00 [queued]>
[2025-07-13T10:36:15.536+0000] {taskinstance.py:2867} INFO - Starting attempt 1 of 1
[2025-07-13T10:36:15.549+0000] {taskinstance.py:2890} INFO - Executing <Task(_PythonDecoratedOperator): split_and_save_data> on 2025-07-13 10:35:00+00:00
[2025-07-13T10:36:15.557+0000] {warnings.py:112} WARNING - /home/airflow/.local/lib/python3.12/site-packages/airflow/task/task_runner/standard_task_runner.py:70: DeprecationWarning: This process (pid=68) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-07-13T10:36:15.558+0000] {standard_task_runner.py:72} INFO - Started process 70 to run task
[2025-07-13T10:36:15.561+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'employee_ingestion_dag', 'split_and_save_data', 'scheduled__2025-07-13T10:35:00+00:00', '--job-id', '8', '--raw', '--subdir', 'DAGS_FOLDER/ingestion_pipeline.py', '--cfg-path', '/tmp/tmpi21fecqv']
[2025-07-13T10:36:15.563+0000] {standard_task_runner.py:105} INFO - Job 8: Subtask split_and_save_data
[2025-07-13T10:36:15.636+0000] {task_command.py:467} INFO - Running <TaskInstance: employee_ingestion_dag.split_and_save_data scheduled__2025-07-13T10:35:00+00:00 [running]> on host c45a93548ada
[2025-07-13T10:36:15.725+0000] {taskinstance.py:3134} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='employee_ingestion_dag' AIRFLOW_CTX_TASK_ID='split_and_save_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-13T10:35:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2025-07-13T10:35:00+00:00'
[2025-07-13T10:36:15.727+0000] {taskinstance.py:732} INFO - ::endgroup::
[2025-07-13T10:36:16.418+0000] {logging_mixin.py:190} INFO - Processing file: employee_data2_with_errors_chunk_18.csv
[2025-07-13T10:36:16.418+0000] {logging_mixin.py:190} INFO - Total rows: 2410, Invalid rows: 0
[2025-07-13T10:36:16.438+0000] {logging_mixin.py:190} INFO - File moved to good_data: /opt/airflow/data/good_data/employee_data2_with_errors_chunk_18.csv
[2025-07-13T10:36:16.440+0000] {logging_mixin.py:190} INFO - Original file removed: /opt/airflow/data/raw_data/employee_data2_with_errors_chunk_18.csv
[2025-07-13T10:36:16.441+0000] {python.py:240} INFO - Done. Returned value was: None
[2025-07-13T10:36:16.451+0000] {taskinstance.py:341} INFO - ::group::Post task execution logs
[2025-07-13T10:36:16.452+0000] {taskinstance.py:353} INFO - Marking task as SUCCESS. dag_id=employee_ingestion_dag, task_id=split_and_save_data, run_id=scheduled__2025-07-13T10:35:00+00:00, execution_date=20250713T103500, start_date=20250713T103615, end_date=20250713T103616
[2025-07-13T10:36:16.539+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-07-13T10:36:16.556+0000] {taskinstance.py:3901} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-07-13T10:36:16.558+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
